src/components/member-profile/contribution/index.js

```diff
@@ -43,7 +50,6 @@ const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
       <div
         className={url && classNames.contributionCard}
         onClick={gotoUrl}
-        onKeyDown={gotoUrl}
         aria-hidden="true"
```

Comment:
removed this as it was causing issue in our super user option, because of this the link was opening up when we had alt key pressed.